cpp/include/cugraph_c/graph.h

```diff
@@ -76,9 +77,55 @@ cugraph_error_code_t cugraph_sg_graph_create(
   cugraph_graph_t** graph,
   cugraph_error_t** error);
 
+/**
+ * @brief     Construct an SG graph
+ *
+ * @param [in]  handle         Handle for accessing resources
+ * @param [in]  properties     Properties of the constructed graph
+ * @param [in]  vertices       Optional device array containing a list of vertex ids
+ *                             (specify NULL if we should create vertex ids from the
+ *                             unique contents of @p src and @p dst)
+ * @param [in]  src            Device array containing the source vertex ids.
+ * @param [in]  dst            Device array containing the destination vertex ids
+ * @param [in]  weights        Device array containing the edge weights.  Note that an unweighted
+ *                             graph can be created by passing weights == NULL.
+ * @param [in]  edge_ids       Device array containing the edge ids for each edge.  Optional
+                               argument that can be NULL if edge ids are not used.
+ * @param [in]  edge_type_ids  Device array containing the edge types for each edge.  Optional
+                               argument that can be NULL if edge types are not used.
+ * @param [in]  store_transposed If true create the graph initially in transposed format
+ * @param [in]  renumber       If true, renumber vertices to make an efficient data structure.
+ *    If false, do not renumber.  Renumbering is required if the vertices are not sequential
+ *    integer values from 0 to num_vertices.
+ * @param [in]  do_expensive_check    If true, do expensive checks to validate the input data
+ *    is consistent with software assumptions.  If false bypass these checks.
+ * @param [in]  properties     Properties of the graph
```

Comment:
properties is defined twice for this function (and also twice for the function below).